java/ql/src/Likely Bugs/I18N/MissingLocaleArgument.qhelp

```diff
@@ -26,8 +26,7 @@ across all locales. For example:</p>
 for different locales. For example, if the default locale is ENGLISH, the function <code>toLowerCase()</code>
 converts a capital <code>I</code> to <code>i</code>; if the default locale is TURKISH, the function 
 <code>toLowerCase()</code> converts a capital <code>I</code> to the Unicode Character "Latin small 
-letter dotless i" (U+0131) (<a href="http://character-code.com/turkish-html-codes.php">Turkish HTML 
```

Comment:
Removed this link because it does not work anymore, is also probably not needed.